src/components/RoleComboBox/RoleComboBox.tsx

```diff
@@ -12,11 +12,12 @@ import { tr } from './RoleComboBox.i18n';
 
 interface RoleComboBoxProps {
     role?: Nullish<Role>;
+    roleName?: string;
```

Comment:
What's this?

Reply:
This for vacancy name: it doesn't have link to role, so it has only role name